xla/service/gpu/fusions/fusions.cc

```diff
@@ -142,15 +142,17 @@ absl::StatusOr<std::unique_ptr<FusionInterface>> GetFusionEmitter(
                          ->config()
                          .debug_options();
   auto check_mlir_emitters = [&](bool check = true) {
+    if (!check) return false;
```

Comment:
This doesn't match the previous logic - check was suppose to only skip the CHECK failure below (because there's no elemental lowering for scatter).